oauth2/.snapshots/TestAuthCodeWithMockStrategy-strategy=opaque-case=0-description=should_pass_request_if_strategy_passes-should_call_refresh_token_hook_if_configured.json

```diff
@@ -32,16 +32,10 @@
     "allowed_top_level_claims": []
   },
   "requester": {
-    "client_id": "app-client",
```

Comment:
This would be a breaking change. I'm not sure how to handle this? Two thoughts come to mind:

1. It probably makes sense to remove the inner duplication (why would someone use the nested keys)
2. The `requester` key is probably more accurate and groups (similar to session) the key correctly

What do you think?